### PR TITLE
CDAP Ambari Service documentation

### PR DESCRIPTION
**Pull request by awholegunch**
Passes [Quick Build](http://builds.cask.co/browse/CDAP-DOB24-4/artifact).

[Ambari Service Page](http://builds.cask.co/artifact/CDAP-DOB24/shared/build-4/Docs-HTML/3.2.0/en/integrations/ambari/index.html)

This will be back-ported to other documentation sets for CDAP versions that can use this service.

### CODE REVIEW

File: cdap-docs/integrations/source/ambari/index.rst [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+========
+
+.. highlight:: console
+
+This section will familiarize you with CDAP's integration with `Apache Ambari
+<https://ambari.apache.org/>`__, the open source provisioning system for `HDP (Hortonworks
+Data Platform) <http://hortonworks.com/>`__.
+
+You can use `Ambari <https://ambari.apache.org>`__ to integrate CDAP into a Hadoop cluster
+by adding the `Cask Ambari Services <https://github.com/caskdata/cdap-ambari-service>`__
```
Comment (wolf31o2) [inline cdap-docs/integrations/source/ambari/index.rst]: CDAP Ambari Service

File: cdap-docs/integrations/source/ambari/index.rst [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+repository to your Ambari server. Once you have restarted your server, you will able to
```
Comment (wolf31o2) [inline cdap-docs/integrations/source/ambari/index.rst]: Remove `repository`

File: cdap-docs/integrations/source/ambari/index.rst [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+      **Ambari Dashboard:** Selecting *CDAP*
+ 
+#. Next, we will assign CDAP services to hosts.
+
+   CDAP consists of 4 daemons:
+ 
+   - **Master:** coordinator service which launches CDAP system services into YARN
+   - **Router:** serves HTTP endpoints for CDAP applications and REST API
+   - **Kafka Server:** used for storing CDAP metrics and CDAP system service log data
+   - **UI:** web interface to CDAP and `Cask Hydrator <http://blog.cask.co/2015/09/cask-hydrator-and-the-future-of-cdap/>`__ (for CDAP 3.2.x installations)
```
Comment (wolf31o2) [inline cdap-docs/integrations/source/ambari/index.rst]: Should this link to the Hydrator docs, instead? I think so.

File: cdap-docs/integrations/source/ambari/index.rst [hunk 4]
```diff
@@ -0,0 +1,278 @@
...
+===========================
+CDAP integration with Ambari is still evolving and improving. Additional features are
+planned for upcoming versions of the CDAP Ambari Service, including a full smoke test of
+CDAP functionality after installation, pre-defined alerts for CDAP services, CDAP
+component High-availability support, select CDAP metrics, support for Kerberos-enabled
+clusters, and integration with the CDAP Authentication Server.
+
+The definition used to create the Ambari service is open source; contributions, issues,
+comments, and suggestions are always welcome and encouraged at the `source code
+repository <https://github.com/caskdata/cdap-ambari-service>`__.
```
Comment (wolf31o2) [inline cdap-docs/integrations/source/ambari/index.rst]: These should go through our main [issue tracker for CDAP](https://issues.cask.co/browse/CDAP) as issues are completely disabled on that repo.

File: cdap-docs/integrations/source/ambari/index.rst [hunk 5]
```diff
@@ -0,0 +1,278 @@
...
+Congratulations! CDAP is now running on your cluster, managed by Ambari.
+
+
+Roadmap and Future Features
+===========================
+CDAP integration with Ambari is still evolving and improving. Additional features are
+planned for upcoming versions of the CDAP Ambari Service, including a full smoke test of
+CDAP functionality after installation, pre-defined alerts for CDAP services, CDAP
+component High-availability support, select CDAP metrics, support for Kerberos-enabled
+clusters, and integration with the CDAP Authentication Server.
```
Comment (wolf31o2) [inline cdap-docs/integrations/source/ambari/index.rst]: I created issues for all of these... CDAP-4105 through CDAP-4110, if you want to link them to actual issues.

**Comment by wolf31o2** [comment]
Couple minor comments.

File: cdap-docs/integrations/source/ambari/index.rst (hunk 1 above)
Comment (awholegunch) [inline cdap-docs/integrations/source/ambari/index.rst]: Fixed.

File: cdap-docs/integrations/source/ambari/index.rst (hunk 2 above)
Comment (awholegunch) [inline cdap-docs/integrations/source/ambari/index.rst]: Fixed.

File: cdap-docs/integrations/source/ambari/index.rst (hunk 4 above)
Comment (awholegunch) [inline cdap-docs/integrations/source/ambari/index.rst]: fixed.

File: cdap-docs/integrations/source/ambari/index.rst (hunk 5 above)
Comment (awholegunch) [inline cdap-docs/integrations/source/ambari/index.rst]: Added and linked.

File: cdap-docs/integrations/source/ambari/index.rst (hunk 3 above)
Comment (awholegunch) [inline cdap-docs/integrations/source/ambari/index.rst]: Added link.

**Comment by awholegunch** [comment]
@wolf31o2 Updated; please take another look.

**Comment by wolf31o2** [comment]
:+1: LGTM

**Pull request merged**